src/main/java/uk/gov/cabinetoffice/csl/service/user/UserAccountService.java

```diff
@@ -22,4 +23,9 @@ public void activateUser(String uid) {
         RegisteredLearnerAccountActivateMessage message = messageMetadataFactory.generateRegisteredLearnerAccountActivateMessage(uid);
         messagingClient.sendMessages(List.of(message));
     }
+
+    public void updateEmail(String uid, String email) {
```

Comment:
We should either clear the cached user in `UserDetailsService` here, or update the email field of the cached user (if they are cached)